### DISCUSSION TITLE
S Meter implementation

### DISCUSSION THREAD

**Discussion by kaefe64** [kaefe64/Arduino_uSDX_Pico_FFT_Proj#8] [Q&A]
I opened this discussion to talk about the S Meter on Arjan-5

**Comment by kaefe64** [comment]
I started the S Meter implementation, but some things are still missing:
 - test the S1..S9++ at display and bar graph. Sometimes I write "space" on display an it does not erase the area like a space, maybe a library bug.
 - include the increase of S when attenuators are activated.
 - find a reliable attenuator to use with my RF Generator to get the RF signal low levels to calibrate the S Meter

**Reply by Trendy73** [reply]
Good job!

**Reply by kaefe64** [reply]
Thank you for the feedback Trendyfu. This helps me keep going...
